src/draw/lv_draw.h

```diff
@@ -165,6 +165,9 @@ struct _lv_layer_t  {
      */
     lv_area_t _clip_area;
 
+    /** Clip radius of the layer. Used for rounded corners */
+    int32_t clip_radius;
```

Comment:
As it can be allied on normal images too (not only layers), I think it should be part of `lv_draw_image_dsc_t`.

Reply:
Done